sources/packages/backend/apps/api/src/constants/class-validator-constants.ts

```diff
@@ -0,0 +1,11 @@
+export const JSON_10KB = 10240;
+export const JSON_20KB = 2048;
```

Comment:
I believe that there is a missing "0" on 2048.

Reply:
Thanks!